src/components/ButtonGroup/ButtonGroup.stories.tsx

```diff
@@ -12,22 +12,34 @@ export default {
     component: ButtonGroupComponent,
     argTypes: {
         size: {
-            options: [ButtonSize.Small, ButtonSize.Medium, ButtonSize.Large],
+            options: ButtonSize,
+            control: { type: 'radio' },
+        },
+        style: {
+            options: ButtonStyle,
+            control: { type: 'radio' },
+        },
+        emphasis: {
+            options: ButtonEmphasis,
             control: { type: 'radio' },
         },
     },
     args: {
         size: ButtonSize.Medium,
+        style: ButtonStyle.Default,
+        emphasis: ButtonEmphasis.Default,
     },
 } as Meta<ButtonGroupProps>;
 
-export const ButtonGroup: Story<ButtonGroupProps> = (args: ButtonGroupProps) => (
+export const ButtonGroup = ({ ...args }) => (
     <ButtonGroupComponent size={args.size}>
```

Comment:
Same here icons should be:
IconAddSimple -> IconPlus
IconCheck -> IconCheckMark
IconReject -> IconCross